### ISSUE TITLE
include additional optional parameters in Add post request

### ISSUE THREAD

**Issue by mozilla-rowvigor** [nevimov/django-easycart#9]
I like the simplicity of django-easycart, but there is one feature I am not sure how to accomplish.

Using the jquery POST add request, I would like to push additional optional options, in addition to the quantity param such as size or color, how would I accomplish that?

**Comment by nevimov** [comment]
Take a look at:
http://django-easycart.readthedocs.io/en/latest/cookbook.html#associating-arbitrary-data-with-cart-items

**Comment by mozilla-rowvigor** [comment]
Yes, I can pass additional arguments on the backend using the add method, however, I can only pass the pk, and quantity arguments with the add POST request. I can't include any additional parameters. Looking at the source code on line 117 of 
https://github.com/nevimov/django-easycart/blob/master/easycart/views.py

I see that the quantity param is the only optional parameter I can pass after the POST AJAX request, every other additional parameter is filtered out.

**Comment by nevimov** [comment]
Yep, looks like you're right. I'll fix the problem within a week.

**Comment by nevimov** [comment]
I've added a quick fix for the problem. See branch 'allow-passing-arbitrary-data'.

However, I've realized that the project needs some serious refactoring. The thing is, I don't use the app anymore and have no time/desire to work on it further. So, I recommend to use another app. I'll put a notice on the main page later.

**Comment by n3storm** [comment]
@mozilla-rowvigor can you ping me if you keep development? We are using it in soon-to-be production

**Comment by mozilla-rowvigor** [comment]
Sure.

**Comment by kamranhossain** [comment]
@mozilla-rowvigor please also ping me. Thanks. :+1:
